CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
It'll take $*, so it's useful for single files, or single suits.
I don't know about `make test-ci` but `vcbuild test-ci` builds first (with Feet of clay), very demotivating.